lte/gateway/c/oai/tasks/mme_app/mme_app_embedded_spgw.c

```diff
@@ -133,6 +138,10 @@ int mme_config_embedded_spgw_parse_opt_line(
     return RETURNerror;
   }
 
+  if (amf_config_parse_file(amf_config_p) != 0) {
```

Comment:
what you are doing is error prone. For instance I recently discovered some bugs in how we sort TAI lists and since parsing logic verbatim copies the same thing, the bug fix over there is not translated over here. Further, you are repeating the same parsing again and again. What I suggest is that if the config from mme is taken verbatim then simply copy the fields from mme_config_p. For things that are not covered by the mme config, parse them using `amf_config_parse_file`.